backup-all.sh

```diff
@@ -3,10 +3,11 @@
 # by piscue
 
 # Setting variables
-backup_path="/home/core/backups"
+backup_path="/opt/backups"
```

Comment:
That could break people's path. If you want to make it generic and accept a variable I will accept the change